src/osd/ReplicatedPG.cc

```diff
@@ -7341,6 +7352,7 @@ void ReplicatedPG::finish_promote(int r, CopyResults *results,
   dout(10) << __func__ << " " << soid << " r=" << r
 	   << " uv" << results->user_version << dendl;
 
+  osd->promotion_ops.put(1);
```

Comment:
given that we unconditionally put() on promote complete, I suggest we take(1) in promote_object(), so that if we missed a s/promote_object/queue_async_promote/, or an explicit promote_object gets added later, it won't break the throttle